csuntechlab/iSTART
# STR-208 -- Wire up the front-end so modules save progress to back-end.

# Description
Wired dashboard routing for proper user groups; Wired get-module-progress on initial login. Wired set-module-progress on completion of module slide.
  
# Testing
1. Ensure after login, the percentages will load up for each module.
2. Check to see if set-module-progress is properly being set for each module.
